src/dbwrapper.cpp

```diff
@@ -71,14 +71,41 @@ class CBitcoinLevelDBLogger : public leveldb::Logger {
     }
 };
 
+constexpr int PermissibleMaxOpenFiles() {
```

Comment:
Add `static`?

Reply:
The `static` specifier isn't necessary because `constexpr` functions are evaluated at compile time. I double checked that `nm ./src/bitcoind | c++filt | grep Permissible` produces no output.